modules/optics/arrow-optics/src/test/kotlin/arrow/optics/std/ListTest.kt

```diff
@@ -35,9 +35,9 @@ class ListTest : UnitSpec() {
 
     testLaws(OptionalLaws.laws(
       optional = ListK.tail(),
-      aGen = Gen.list(Gen.int()),
-      bGen = Gen.list(Gen.int()),
-      funcGen = genFunctionAToB(Gen.list(Gen.int())),
+      aGen = genNonEmptyList(Gen.int()).map { it.all },
```

Comment:
?

Reply:
set is not consistent with modify on a empty list due to:
set add a value to an empty list
modify does nothing over an empty list

Reply:
Did these test break in this PR?

Reply:
Yes, sorry